tests/qos/files/qos_params.pac.yaml

```diff
@@ -149,6 +149,15 @@ qos_params:
                     pkts_num_margin: 4
                     packet_size: 8140
                     cell_size: 8192
+                lossy_queue_voq_2:
+                    dscp: 8
+                    ecn: 1
+                    pg: 0
+                    flow_config: shared
+                    pkts_num_trig_egr_drp: 10241
+                    pkts_num_margin: 4
+                    packet_size: 8140
```

Comment:
we will for this file to be merged and then we will update 202205 PR with this change: https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsonic-net%2Fsonic-mgmt%2Fpull%2F10222%23issuecomment-1760486230&data=05%7C01%7CAbhishek.Dosi%40microsoft.com%7C505bbd8078464fca667b08dbcb775097%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638327485541613350%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Z1WA1HZyK6L1vGV5n%2BkAN0AeywoxMCRZb30vU%2BKpzng%3D&reserved=0

Reply:
Added in 202205 PR https://github.com/sonic-net/sonic-mgmt/pull/10222